lib/resources/crontab.rb

```diff
@@ -25,21 +24,26 @@ class Crontab < Inspec.resource(1)
       describe crontab.where { command =~ /a partial command string/ } do
         its('entries.length') { should cmp 1 }
       end
+
+      describe crontab('/etc/cron.d/some_crontab') do
+        its('commands') { should include '/path/to/some/script' }
+      end
     "
 
     attr_reader :params
 
     include CommentParser
 
-    def initialize(user = nil)
-      @user   = user
+    def initialize(destination = nil)
+      @destination = destination
       @params = read_crontab
 
       return skip_resource 'The `crontab` resource is not supported on your OS.' unless inspec.os.unix?
     end
 
     def read_crontab
-      inspec.command(crontab_cmd).stdout.lines.map { |l| parse_crontab_line(l) }.compact
+      ct = path? ? inspec.file(@destination).content : inspec.command(crontab_cmd).stdout
+      ct.lines.map { |l| parse_crontab_line(l) }.compact
```

Comment:
Non-user crontabs, such as those in `/etc/cron.d` follow a different format. Specifically, before the command, there is another field for the user via which the command should be run. We'd need to adapt the resource to handle that new format for non-user crontabs.